testharness.go

```diff
@@ -337,45 +337,50 @@ func min(number []int) int {
 
 func checkURLs(urls []string) (bool, error) {
 
-	robotsAddress := ""
 	allAllowed := true
+	robotsAddress := ""
 	robots, err := robotstxt.FromString("User-agent: *\nDisallow:") //init robots
 	if err != nil {
 		return false, err
 	}
 
-	for _, u1 := range urls {
+	if WhiteListed {
 
-		u, err := url.Parse(u1)
-		if err != nil {
-			return false, err
-		}
+		fmt.Printf("THIS PROVIDER IS WHITELISTED, IGNORING ROBOTS.TXT CHECK\n")
```

Comment:
you know you can just do:

```go
fmt.Println("THIS STRING GETS A LINE BREAK AUTOMATICALLY")
```

There isn't a version of this like the `Printf` ones for interpolation, but if you have no variables to munge into the string you can just do the above.